<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardUndistortionFrame</class>
 <widget class="QFrame" name="WizardUndistortionFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameModifyUndistortion">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBoxMouseClickMode">
        <property name="title">
         <string>mouseclick mode</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButtonMouseClickOutlier">
           <property name="text">
            <string>toggle outlier</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonMouseClickNone">
           <property name="text">
            <string>None</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButtonMouseClickCenter">
           <property name="text">
            <string>set center</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBoxVisualization">
        <property name="title">
         <string>visualization</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>165</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>image</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>distorted</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>undistorted</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>distortion map (scaled angles)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>distortion map (angles)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>distortion map (distance)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>distortion map (horizontal displacement)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>distortion map (vertical displacement)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>165</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>points   </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxPoints">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>none</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>detected (distorted)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>grid points (distorted)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>grid points (undistorted)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>reference points (undistorted)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButtonRemoveOutlierAutomatically">
        <property name="text">
         <string>remove outlier automatically</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
